sdk/bpf/scripts/install.sh

```diff
@@ -22,12 +22,10 @@ download() {
   wget "${args[@]}"
 }
 
-# Install or upgrade xargo
+# Install xargo
 (
-  cargo install cargo-update
-  cargo install-update-config --version =0.3.19 xargo
-  set -e
-  cargo install-update -i xargo
+  set -ex
+  cargo +"${rust_stable:-}" install xargo
```

Comment:
I'd just go with a simple approach:
```suggestion
  if [[ -n $rust_stable ]]; then
    cargo +"${rust_stable:-}" install xargo
  else
    cargo install xargo
  fi
```

Reply:
shellcheck does not like this since it can't tell if rust_stable is defined (rust_stable should probably be in all caps) and would rather not have to clutter up with a spellcheck "allow" statement